utils_cv/tracking/model.py

```diff
@@ -281,47 +280,80 @@ def save(self, path) -> None:
 
         save_model(path, self.epoch, self.model, self.optimizer)
         print(f"Model saved to {path}")
-
-    def evaluate(self,
-                 results: Dict[int, List[TrackingBbox]],
-                 gt_root_path: str) -> str:
+                
+    def savetxt_results(self,
```

Comment:
If I am not mistaken, this looks like a copy&paste of an old function which Karine already modified and checked in. @karineip